### PR TITLE
EPA source update

### PR DESCRIPTION
**Pull request by maryarbab**
Is this a new addition or an update to an existing factor?

Update

What category or categories does it affect?

Multiple categories  

What?

Updates to the year, source link, value, and description fields.

Why?

To link to the most recent EPA and eGrid sources.

### CODE REVIEW

**Review by greenercode — DISMISSED** [review]

File: OpenEmissionFactorsDB.csv [hunk 1]
```diff
@@ -1095,8 +1095,8 @@ Materials,Mined Materials,mined_materials-type_copper_nickel_lead_zinc,Copper/ni
...
+Materials,Mining,mining-type_other_support_activities_for_mining,Other support activities for mining,kg-CO2e/USD,0.613,,EPA,2020,US,2021/10/12,Emission intensity of supply chain (with margins i.e. cradle to shelf) in US dollars spend on: other support activities for mining. This factor is representative of the described commodity (equivalent to a goods or services category) and was calculated from 2016 data applying IPCC 4th Assessment Report conversion factors.,https://edg.epa.gov/metadata/catalog/search/resource/details.page?uuid=https://doi.org/10.23719/1517796,cradle_to_shelf
+Materials,Mining,mining-type_well_drilling,Well drilling,kg-CO2e/USD,0.543,,EPA,2020,US,2021/10/12,Emission intensity of supply chain (with margins i.e. cradle to shelf) in US dollars spend on: well drilling. This factor is representative of the described commodity (equivalent to a goods or services category) and was calculated from 2016 data applying IPCC 4th Assessment Report conversion factors.,https://edg.epa.gov/metadata/catalog/search/resource/details.page?uuid=https://doi.org/10.23719/1517796,cradle_to_shelf
```
Comment (patstirling) [inline OpenEmissionFactorsDB.csv]: remove to avoid conflicts

File: OpenEmissionFactorsDB.csv [hunk 2]
```diff
@@ -356,139 +356,139 @@ Energy,Electricity,electricity-energy_source_grid_mix,Grid mix,kg-CO2e/kWh,0.84,
...
-Energy,Electricity,electricity-energy_source_grid_mix,Grid mix,kg-CO2e/kWh,0.79312,,EPA,2018,US-IN,2021/07/23,Emission intensity of grid mix as reported. Retrieved from EPA eGrid 2020. Includes generation factor.,https://www.epa.gov/egrid,electricity_generation
-Energy,Electricity,electricity-energy_source_grid_mix,Grid mix,kg-CO2e/kWh,0.77695,,EPA,2018,US-MO,2021/07/23,Emission intensity of grid mix as reported. Retrieved from EPA eGrid 2020. Includes generation factor.,https://www.epa.gov/egrid,electricity_generation
-Energy,Electricity,electricity-energy_source_grid_mix,Grid mix,kg-CO2e/kWh,0.37128,,EPA,2018,US-IL,2021/07/23,Emission intensity of grid mix as reported. Retrieved from EPA eGrid 2020. Includes generation factor.,https://www.epa.gov/egrid,electricity_generation
+Energy,Electricity,electricity-energy_source_grid_mix,Grid mix,kg-CO2e/kWh,0.603764064,,EPA,2021,US-CO,2021/07/23,"Emission intensity of grid mix as reported. Total output emission rate, available in eGRID2019 Data File (XLSX). ",https://www.epa.gov/egrid/download-data,electricity_generation
```
Comment (patstirling) [inline OpenEmissionFactorsDB.csv]: Replace "available in" with "retrieved from" throughout

**Review by patstirling — CHANGES_REQUESTED** [review]

File: OpenEmissionFactorsDB.csv (hunk 2 above)
[the remark this answers is not in the document]
Comment (maryarbab) [reply OpenEmissionFactorsDB.csv]: Let's make this change as a separate PR as it will impact multiple sources.

File: OpenEmissionFactorsDB.csv [hunk 3]
```diff
@@ -356,139 +356,139 @@ Energy,Electricity,electricity-energy_source_grid_mix,Grid mix,kg-CO2e/kWh,0.84,
...
-Energy,Electricity,electricity-energy_source_grid_mix,Grid mix,kg-CO2e/kWh,0.22777,,EPA,2018,US-NJ,2021/07/23,Emission intensity of grid mix as reported. Retrieved from EPA eGrid 2020. Includes generation factor.,https://www.epa.gov/egrid,electricity_generation
-Energy,Electricity,electricity-energy_source_grid_mix,Grid mix,kg-CO2e/kWh,0.52886,,EPA,2018,US-MT,2021/07/23,Emission intensity of grid mix as reported. Retrieved from EPA eGrid 2020. Includes generation factor.,https://www.epa.gov/egrid,electricity_generation
+Energy,Electricity,electricity-energy_source_grid_mix,Grid mix,kg-CO2e/kWh,0.247292915,,EPA,2021,US-NJ,2021/07/23,Emission intensity of grid mix as reported. Total output emission rate. Available in eGRID2019 Data File (XLSX). ,https://www.epa.gov/egrid/download-data,electricity_generation
+Energy,Electricity,electricity-energy_source_grid_mix,Grid mix,kg-CO2e/kWh,0.572764226,,EPA,2021,US-MT,2021/07/23,Emission intensity of grid mix as reported. Total output emission rate. Available in eGRID2019 Data File (XLSX). ,https://www.epa.gov/egrid/download-data,electricity_generation
```
Comment (AnastasiaLobanova) [inline OpenEmissionFactorsDB.csv]: Here I have the same, general comment - was this data wrong? Or it was replaced with the latest EF? I think we should discuss if we want to keep the historic data?

**Review by AnastasiaLobanova — APPROVED** [review]

**Review by patstirling — DISMISSED** [review]

**Review by patstirling — APPROVED** [review]

**Pull request merged**